apt-faster/action.yaml

```diff
@@ -0,0 +1,111 @@
+# Copyright 2025 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+name: "apt-faster"
+description: "Make apt go faster"
+
+runs:
+  using: composite
+  steps:
+    - name: Write apt-get wrapper
+      shell: bash
+      run: |
+        sudo tee /usr/bin/apt-get.wrap >/dev/null <<"EOF"
+        #!/bin/sh
+        a=" $* "
+        emd="eatmydata"
+        if ! command -v eatmydata >/dev/null 2>&1; then
+           emd=""
+           if [ "$(id -u)" = "0" ] &&
+              [ "${a#* install }" != "$a" -o "${a#* upgrade }" != "$a" -o \
+                "${a#* dist-upgrade }" != "$a" ]; then
+             # we are root, this is install, so install emd
+             echo "=== first installing eatmydata ===" 1>&2
+             out=$(DEBIAN_FRONTEND=noninteractive "$0.distrib" install \
+                   --quiet --assume-yes eatmydata 2>&1)
+             ret=$?
+             if [ $ret -ne 0 ]; then
+                echo "FAILED: $out" 1>&2;
+                echo "failed to install libeatmydata."
+                exit $ret
+             fi
+             emd="eatmydata"
+          fi
+        fi 1>&2 </dev/null
+        exec $emd "apt-get.distrib" "$@"
+        EOF
+
+        sudo chmod 755 /usr/bin/apt-get.wrap
+
+        command -v eatmydata >/dev/null &&
+          echo "eatmydata was present" ||
+          echo "eatmydata not installed"
+
+    - name: replace apt-get
+      shell: bash
+      run: |
+        [ -e "/usr/bin/apt-get.distrib" ] ||
+          sudo dpkg-divert --local --rename --add /usr/bin/apt-get
+        sudo ln -sf apt-get.wrap /usr/bin/apt-get
```

Comment:
`apt-get.apt-faster` might be a useful breadcrumb than `apt-get.wrap`

Reply:
done